ember-phone-input/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "extends": "@tsconfig/ember/tsconfig.json",
+  "include": ["src/**/*", "unpublished-development-types/**/*"],
+  "glint": {
+    "environment": "ember-loose"
+  },
+  "compilerOptions": {
+    "allowJs": true,
+    "declarationDir": "declarations",
+    "skipLibCheck": true
+  }
+}
```

Are we not missing an extra line at the end of the file?

```suggestion
}

```

---

Yep, fixed